cargo-near/src/commands/new/mod.rs

```diff
@@ -148,6 +91,89 @@ impl NewContext {
     }
 }
 
+#[tracing::instrument(
+    target = "tracing_instrument",
+    name = "The process of executing",
+    skip_all
+)]
+fn execute_git_commands(project_dir: &std::path::Path) -> near_cli_rs::CliResult {
+    tracing::Span::current().pb_set_message("`git` commands ...");
```

Comment:
this one doesn't get displayed  on `--teach-me`, but we probably can figure this out in subsequent prs